### DISCUSSION TITLE
Coordinate System

### DISCUSSION THREAD

**Discussion by ElfWitch** [recastnavigation/recastnavigation#697] [Q&A] [answered]
I can't find a formal mention of recast / detour's coordinate system anywhere. This might be something useful to add to the FAQ, but it would also be nice if it was mentioned here.

**Comment by ElfWitch** [comment, marked as answer]
For future reference: "In general, Recast expects **clockwise-winding** triangles and the coordinate system Recast uses is **right-handed** and **Y-up**" - https://groups.google.com/g/recastnavigation/c/tyjIk9MEJe8/m/psWrPGH2AQAJ

**Reply by grahamboree** [reply]
Thanks!! I'll add this to the FAQ.
